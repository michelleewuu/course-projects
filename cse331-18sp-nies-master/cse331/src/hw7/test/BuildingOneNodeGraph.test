# Test building graph with only one Node.

# Load oneNode.tsv
LoadGraph oneNodeGraph oneNode.tsv

# Print the nodes in the graph and the edges starting from onlyNode
ListNodes oneNodeGraph
ListChildren oneNodeGraph onlyNode