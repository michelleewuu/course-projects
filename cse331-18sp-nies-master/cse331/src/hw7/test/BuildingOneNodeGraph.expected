# Test building graph with only one Node.

# Load oneNode.tsv
loaded graph oneNodeGraph

# Print the nodes in the graph and the edges starting from onlyNode
oneNodeGraph contains: onlyNode
the children of onlyNode in oneNodeGraph are: